components/molecules/OptionalListField.js

```diff
@@ -147,4 +156,9 @@ OptionalListField.propTypes = {
    * legend text for the list items
    */
   listLabel: PropTypes.string,
+
+  /**
+   * sort the children being passed in based on labels
+   */
+  sortedChildren: PropTypes.func,
```

Comment:
You don't need to add this since `sortedChildren` is not being passed into the component.